Update ws4py port options

sonicaj opened the pull request:
This commit updates ws4py port options for FN 12.

justinclift commented:
This is a reworked version of the freenas-build README.me to be more user friendly, and also point at the build guide in the wiki as first reference place.

justinclift commented:
As a thought, the issue for this on bugs.freenas.org can probably be closed now too:

&nbsp; &nbsp; https://bugs.freenas.org/issues/15397

sonicaj committed:
Update ws4py port options

This commit updates ws4py port options for FN 12.

william-gr reviewed (approved):
Lets also unset GEVENT here too please since we dont use

sonicaj committed:
Unset gevent for ws4py

In this commit we unset gevent for ws4py as that is not used by us and not needed in the build.
